### PR TITLE
Use initial version of Starcounter/uniform.css@0.1.0 /uni-date-picker

### PR DESCRIPTION
**Pull request by tomalec**
#271

I created new PR in place of https://github.com/Starcounter/KitchenSink/pull/272 so it would be easier to review

### CODE REVIEW

**Comment by tomalec** [comment]
Manual tests are passing, but I'm unable to make selenium tests pass

**Comment by tomalec** [comment]
@kegor Could you help me with 
```
OpenQA.Selenium.NoSuchElementException : Could not find element by: By.TagName: input[type='date']
   at OpenQA.Selenium.Support.PageObjects.DefaultElementLocator.LocateElement(IEnumerable`1 bys)
   at OpenQA.Selenium.Support.PageObjects.WebElementProxy.get_Element()
   at OpenQA.Selenium.Support.PageObjects.WebElementProxy.Invoke(IMessage msg)
   at System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   at OpenQA.Selenium.IWebElement.Clear()
   at KitchenSink.Tests.Test.DatepickerPageTest.DatepickerPage_SelectDate() in C:\TeamCity\BuildAgent\work\build-302\KitchenSink\test\KitchenSink.Tests\Test\DatepickerPageTest.cs:line 35
```

at https://github.com/Starcounter/KitchenSink/pull/281/commits/7eb6e2a8a3afd99b0248087c8beac16a7b3e77cd#diff-593d65972cd3a02fbbb0edfa2968c622R33

**Comment by alshakero** [comment]
Exactly. `input[type='date']` is not a tag name. It's a CSS selector.

File: src/KitchenSink/wwwroot/KitchenSink/DatepickerPage.html [hunk 1]
```diff
@@ -36,7 +36,10 @@ <h1 slot="kitchensink/datepicker-heading">Datepicker</h1>
             <div class="uni-card">
                 <slot name="kitchensink/datepicker-description"></slot>
                 <div class="uni-label-control">
-                    <slot name="kitchensink/datepicker-date-picker"></slot>
+                    <uni-date-picker slot="kitchensink/datepicker-date-picker">
+                        <slot name="kitchensink/datepicker-label"></slot>
+                        <slot name="kitchensink/datepicker-date"></slot>
+                    </uni-date-picker>
```
Comment (warpech) [inline src/KitchenSink/wwwroot/KitchenSink/DatepickerPage.html]: Please slot the label outside of `uni-date-picker`

**Comment by tomalec** [comment]
It seems I now fall down to date formatting issue, or something I cannot debug, as I have a different locale on my local machine :| Do you have any experience with debugging tests directly on TC, as I have different results on local env?

**Comment by tomalec** [comment]
@joozek78 Could you help us? It seems that even though https://teamcity.starcounter.org/viewLog.html?buildId=53890&buildTypeId=Apps_KitchenSink_KitchenSinkMaster24branchWithStarcounterDevelopNightly&tab=buildParameters says there is Starcounter 2.4.0.5922 running, which contains `C:\Program Files\Starcounter\ClientFiles\StaticFiles\sys\uniform.css\components\uni-date-picker` there is 404 for it, and there is the older version of `uniform.css`

**Comment by warpech** [comment]
12 tests are failing on TC, including two tests from this PR: https://teamcity.starcounter.org/viewLog.html?buildId=53926&tab=buildResultsDiv&buildTypeId=Apps_KitchenSink_KitchenSinkMaster24branchWithStarcounterDevelopNightly

Frankly speaking, most of the problem comes from TC, because on `master-2.4` branch also 12 tests are failing: https://teamcity.starcounter.org/viewLog.html?buildId=53934&tab=buildResultsDiv&buildTypeId=Apps_KitchenSink_KitchenSinkMaster24branchWithStarcounterDevelopNightly

If @PatrykSzwer agrees, I think this can be merged and the tests should be fixed as high priority in a separate PR.

**Comment by tomalec** [comment]
@alshakero  @ubbeK do you know why the artifact of [SC Build 5922  on TC](https://teamcity.starcounter.org/viewLog.html?buildId=53857&buildTypeId=Starcounter_DevelopNightlyWindows&tab=buildParameters)
Contains theoretically [SCF@11835645d9c](https://github.com/Starcounter/StarcounterClientFiles/commit/11835645d9c6b8b5408883e430c7dd97f491b804) => uniform.css@0.1.0, but in fact artifact contain older version:
 https://teamcity.starcounter.org/viewLog.html?buildId=53857&buildTypeId=Starcounter_DevelopNightlyWindows&tab=artifacts#!gh92magqjvny,-12zdvm3agtwzx,8bi8n9irhd1l,-131339wqot1fs,-aodgpuo3djj1,1036qbcpznyq3,1b2dmbuosxu6w,-aemrv8thl19i,-718sx1mh0if8,-ugi0t8wpvpbu


What is even stranger, downloadable build at http://downloads.starcounter.com/download/Starcounter/Develop/latest contains 0.1.0

**Comment by tomalec** [comment]
> 12 tests are failing on TC, including two tests from this PR 

Those tests are failing as Starcounter version used for testing for some reason does not contain latest StarcounterClientFiles

**Comment by ubbeK** [comment]
https://github.com/Starcounter/level1/blob/develop/src/BuildSystem/ClientFiles/ClientFiles.csproj#L10

That version is used, add wildcard if you prefer, `3.4.*`.

**Comment by tomalec** [comment]
Then, what is the meaning of `build.vcs.number.GitStarcounterClientFiles` at https://teamcity.starcounter.org/viewLog.html?buildId=53857&buildTypeId=Starcounter_DevelopNightlyWindows&tab=buildParameters
as it points to something from `3.5.0`.

Also why installer at downloads page uses `3.5.0`?

**Comment by kegor** [comment]
btw, @PatrykSzwer maybe be we can permanently remove parallel tests executing for KitchenSink? This affects only execution time or something else?

**Comment by tomalec** [comment]
After changes in TC builds and tests are passing https://teamcity.starcounter.org/viewLog.html?buildId=54073&.
@warpech  @PatrykSzwer  is it good to merge?

**Comment by PatrykSzwer** [comment]
>btw, @PatrykSzwer maybe be we can permanently remove parallel tests executing for KitchenSink? This affects only execution time or something else?

Yes and no.
Removing parallel executing will extend tests execution significantly. On the other hand, I think It's still better than flaky tests caused by Team City lack of performance. Check the time difference between parallel execution and non-parallel one.

**Pull request merged**